src/triggers.js

```diff
@@ -733,6 +733,118 @@ function builtInTriggerValidator(options, request) {
       }
     }
   }
+  const aclOptions = options.setACL;
+  if (aclOptions && request.object && functionName === 'beforeSave.BeforeSave') {
```

Comment:
Maybe some error message if someone try to setACL for other function that not beforeSave?

Reply:
Can do!